src/pages/settings/InitialSettingsPage.js

```diff
@@ -164,187 +131,165 @@ function InitialSettingsPage(props) {
+     */
+    const getMenuItemsSection = useCallback(
+        (menuItemsData) => {
+            /**
+             * @param {Boolean} isPaymentItem whether the item being rendered is the payments menu item
+             * @returns {String|undefined} the user's wallet balance
+             */
+            const getWalletBalance = (isPaymentItem) => (isPaymentItem ? CurrencyUtils.convertToDisplayString(props.userWallet.currentBalance) : undefined);
+            return (
+                <View style={[menuItemsData.sectionStyle, styles.pb4, styles.mh3]}>
+                    <Text style={styles.sectionTitle}>{translate(menuItemsData.sectionTranslationKey)}</Text>
+                    {_.map(menuItemsData.items, (item, index) => {
+                        const keyTitle = item.translationKey ? translate(item.translationKey) : item.title;
+                        const isPaymentItem = item.translationKey === 'common.wallet';
+
+                        return (
+                            <MenuItem
+                                key={`${keyTitle}_${index}`}
+                                wrapperStyle={styles.sectionMenuItem}
+                                title={keyTitle}
+                                icon={item.icon}
+                                iconType={item.iconType}
+                                disabled={isExecuting}
+                                onPress={singleExecution(() => {
+                                    if (item.action) {
+                                        item.action();
+                                    } else {
+                                        waitForNavigate(() => {
+                                            Navigation.navigate(item.routeName);
+                                        })();
+                                    }
+                                })}
+                                iconStyles={item.iconStyles}
+                                badgeText={getWalletBalance(isPaymentItem)}
+                                fallbackIcon={item.fallbackIcon}
+                                brickRoadIndicator={item.brickRoadIndicator}
+                                floatRightAvatars={item.floatRightAvatars}
+                                shouldStackHorizontally={item.shouldStackHorizontally}
+                                floatRightAvatarSize={item.avatarSize}
+                                ref={popoverAnchor}
+                                shouldBlockSelection={Boolean(item.link)}
+                                onSecondaryInteraction={
+                                    !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor.current) : undefined
+                                }
+                                focused={activeRoute && item.routeName && activeRoute.toLowerCase().replaceAll('_', '') === item.routeName.toLowerCase().replaceAll('/', '')}
```

Comment:
There is an inconsistency with the use of activeRoute. In `InitialSettingsPage` we use route names where in `WorkspaceInitialPage` we use screen names. I think we are interested in the screen names since the useActiveRoute returns the screen name.

Let's fix this inconsistency and clear the logic here to be just:
```js
focused={item.routeName && item.routeName === activeRoute}
```